### ISSUE TITLE
Visibility & Color Helper Classes

### ISSUE THREAD

**Issue by sn0rcha** [lusaxweb/vuesax#212]
**Is your feature request related to a problem? Please describe.**
Missing the ability to hide and show elements on different devices (Desktop / Tablet / Mobile). Missing color based helper classes (Background color / Text Color based off the theme)

**Describe the solution you'd like**
Similar to other frameworks - "hide-sm" "show-for-lg" "text-primary" "bg-primary" etc

**Describe alternatives you've considered**
Coding my own

**Additional context**
Framework is awesome so far, great job!

**Comment by gringolalia** [comment]
Agreed: very nice! For spacing and alignment and all that stuff I had success using a [Bootstrap 4 utilities-only stylesheet](https://github.com/rishabh-rajgarhia/bootstrap-4-utilities).
